### PR TITLE
Update cards-actions.md

### PR DESCRIPTION
**Pull request by dclaux**
Fix inaccurate document about imBack support in Adaptive Cards

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/task-modules-and-cards/cards/cards-actions.md](https://github.com/MicrosoftDocs/msteams-docs/blob/fe5ff0825d543e9eeec053abb9faa8a469b038da/msteams-platform%2Ftask-modules-and-cards%2Fcards%2Fcards-actions.md) | 92 | 13 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/3456ae9f-ba38-46d8-a0ae-b3fc08a2a17b) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by heyitsaamir — APPROVED** [review]

**Review by deekshanya — APPROVED** [review]

**Review by heyitsaamir — APPROVED** [review]

**Review by surbhigupta12 — DISMISSED** [review]

File: msteams-platform/task-modules-and-cards/cards/cards-actions.md [hunk 1]
```diff
@@ -614,7 +614,7 @@ The following code shows an example of Adaptive Cards with `messageBack` action:
 To include an `imBack` action with an Adaptive Card, include the following details in the `msteams` object:
 
 > [!NOTE]
-> You can include additional hidden properties in the `data` object, if required.
+> The `value` field is a simple string. There is no support for formatting or hidden characters.
```
Comment (v-ypalikila) [inline msteams-platform/task-modules-and-cards/cards/cards-actions.md:617]:
```suggestion
> The `value` field is a simple string that doesn’t support formatting or hidden characters.
```

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/task-modules-and-cards/cards/cards-actions.md](https://github.com/MicrosoftDocs/msteams-docs/blob/8dc21cccd6266c0011c7ff65023bc9c15cbb99f7/msteams-platform%2Ftask-modules-and-cards%2Fcards%2Fcards-actions.md) | 92 | 13 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/d0576869-3dda-483f-a0cc-3a6fae1ef3fc) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by surbhigupta12 — APPROVED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/task-modules-and-cards/cards/cards-actions.md](https://github.com/MicrosoftDocs/msteams-docs/blob/33c50f4a2e17db6a47bbc6bb103f989e9cc4d6ba/msteams-platform%2Ftask-modules-and-cards%2Fcards%2Fcards-actions.md) | 92 | 13 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/2054457f-b819-46c4-827d-af272910ce2a) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Pull request merged**